test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -619,6 +618,18 @@ class test_ucp_sockaddr : public ucp_test {
         EXPECT_EQ(1ul, e.get_err_num_rejected());
     }
 
+    template <typename Func>
+    void wait_progress(Func func)
+    {
+        ucs_time_t deadline = ucs::get_deadline();
+
+        while (!func() && (ucs_get_time() < deadline)) {
+            progress();
+        };
+
+        EXPECT_GT(deadline, ucs_get_time());
```

Comment:
can remove space lines

Reply:
ok, removed just the first one
IIRC we need a space line after code block within {} brackets